manifests/init.pp

```diff
@@ -49,6 +50,11 @@
   $service_ensure       = $::dns::params::service_ensure,
   $service_enable       = $::dns::params::service_enable,
   $additional_options   = $::dns::params::additional_options,
+  $transfert_source     = $::dns::params::transfert_source,
+  $transfert_source_port= $::dns::params::transfert_source_port,
```

Comment:
Would it be better if the user would provide the (optional) port as part of the transfer source?